scripts/Tools/testreporter.py

```diff
@@ -0,0 +1,240 @@
+#!/usr/bin/env python
+
+"""
+
+Simple script to populate CESM test database with test results.  This can be 
+run with out any knowledge of CIME.
+
+"""
+
+import os
```

Comment:
I wanted to keep testreporter independent of CIME for the times I populate the testdb later, after I've removed the source code.  Granted this isn't going to happen very often, if ever.  But I still have the problem of finding the CIMEROOT.  I'll either have to get CIMEROOT out of software_environment.txt, or parse env_case.xml.  Now that I think about it, I could set it up like cs.status.  cs.status copies the template to the testing directory, that then calls the cs.status script in CIME.